dialogflow/api/DialogflowSamples/ContextManagement.cs

```diff
@@ -1,128 +0,0 @@
-﻿// Copyright(c) 2018 Google Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License"); you may not
-// use this file except in compliance with the License. You may obtain a copy of
-// the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-// License for the specific language governing permissions and limitations under
-// the License.
-
-using System;
-using System.Linq;
-using System.Collections.Generic;
-using CommandLine;
-using Google.Cloud.Dialogflow.V2;
-
-namespace GoogleCloudSamples
-{
-    // Samples demonstrating how to Create, List, Delete Contexts
-    public class ContextManagement
-    {
-        public static void RegisterCommands(VerbMap<object> verbMap)

```

Comment:
I bet this method is being called from somewhere, probably Program.cs, and similar for similar methods on the other deleted samples. Have you made sure that the remaining code still builds after deleting these?

Reply:
Thank you for your patience, @amanda-tarafa , and sorry for the delayed response.  Unfortunately I'm running this project across multiple samples repositories (for all languages), and each time requires learning the language/test architecture. I will dig into this and figure out which method references I broke - I'm sure you're right.

Reply:
Don't worry, take your time. And let me know if I can help with something.